### ISSUE TITLE
Leagues with multiple Regions

### ISSUE THREAD

**Issue by NexxusDrako** [NexxusDrako/leagion-scorinator#11]
In order to facilitate promotion and relegation, we first need to create leagues (or levels, tiers, etc) and split them into multiple divisasi (regions, paralell leagues)

Quote from Rejis:

> Actually, the first thing we need to do is to have leagues and varying amoungs of divisasi. So for the rejistanian league it would probably look like this:
> =leagues 3
> =regions 1 2 4
> =relegate 2
> =relplayoff

We will still advocate the use of divisasi if auto-relegation is not used, if it is, then we will use leagues.

With regions, if say "=regions 1 2 4" and "=relegate 2" then from the 1st league, one would go to the 1st region and one to the 2nd region. Likewise, in the 2nd league, the two that go down from region 1 go to 1 and 2, while region 2's relegated go to 3 and 4.

Also, we should add to reports the words "League X, Region X". which will also make this an option for those wishing to use those naming conventions.
##

**Comment by NexxusDrako** [comment]
I do think this would require a fair amount of code restructure, so I feel I should move it to the backlog for now and work on smaller things for a bit.

**Comment by Rejis** [comment]
Just to explain just like the German Regionalliga has Nord and Süd, the H2SR has Nahtaj and Nahsua, So one team relegates into the Nahtaj and one into the Nahsua...

**Comment by NexxusDrako** [comment]
Yes, that is true. I should have mentioned that, but failed to do so. I'll update the OP.
